pkg/leakybucket/manager.go

```diff
@@ -500,13 +508,25 @@ func PourItemToHolders(parsed types.Event, holders []BucketFactory, buckets *Buc
 				holder.logger.Errorf("unexpected non-bool return : %T", output)
 				log.Fatalf("Filter issue")
 			}
+
+			if condition {
+				holder.logger.Debugf("eval(%s) = TRUE ", holder.Filter)
```

Comment:
can we switch the debug print and the proper logic code, to avoid the code repetition ?